### PR TITLE
CP-4886: Add promition to release to pypi.cloudzero.com after master build

### PR DESCRIPTION
**Pull request by boboakes**
## Description of the change

Add the semaphore yaml configurations to deploy the newer libraries to pypi.cloudzero.com after a successful build of the master branch.

## Type of change
- [ ] Bug fix
- [x] New feature

### Development
- [ ] All changed code has 80% unit test coverage
- [ ] All changed code has been automatically (smoke test or otherwise) or manually verified in `alfa` (or with a cross namespace setup, e.g. developer namespace for this feature, pointing at shared `alfa` resources)

### Code review 
- [x]  This pull request has a title that includes the ticket # and a short useful summary, e.g. `CP-4051: Create TEMPLATE Feature Repo`.

### CODE REVIEW

**Review by Tankanow — CHANGES_REQUESTED** [review]
@boboakes, this is not right, pyfaaster is actually a public repo published to the public pypi, I think we can use the `make upload` target. (sorry for the short PR comment, I'm on a call)

File: .semaphore/release.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      prologue:
+        commands:
+          - sem-version python 3.8
+          - checkout
+          - cache restore dev-requirements-$SEMAPHORE_GIT_BRANCH-$(checksum requirements-dev.txt),requirements
+          - make init
+      jobs:
+        - name: Upload
+          commands:
+            - make upload
```
Comment (Tankanow) [inline .semaphore/release.yml:19]: MUST: I think we need those `TWINE` credentials for the push `make TWINE_USERNAME=${PYPI_USERNAME} TWINE_PASSWORD=${PYPI_PASSWORD} upload`

Comment (Tankanow) [reply .semaphore/release.yml:19]: You can use the semaphore secret `PyPi Deployment`, which will put `PYPI_USERNAME` and `PYPI_PASSWORD` into the environment. So you can use the `make` command from the previous comment as is.

**Review by Tankanow — CHANGES_REQUESTED** [review]
Should change the PR title; also 1 change to make

File: .semaphore/release.yml (hunk 1 above)
Comment (boboakes) [reply .semaphore/release.yml]: Done.

**Review by Tankanow — APPROVED** [review]

**Pull request merged**